MAUI/Polar-Charts/Getting-Started.md

```diff
@@ -0,0 +1,497 @@
+        <Grid>
+            <chart:SfPolarChart/>
+        </Grid>
+    </ContentPage>
+    
+{% endhighlight %}
+
+{% highlight C# %}
+
+    using Syncfusion.Maui.Charts;
+    namespace ChartGettingStarted
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();           
+                SfPolarChart chart = new SfPolarChart(); 
+            }
+        }   
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Register the handler
+
+Syncfusion.Maui.Core NuGet package is a dependent package for all Syncfusion controls in .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
+
+{% highlight C# %}
+
+    using Microsoft.Maui;
+    using Microsoft.Maui.Hosting;
+    using Microsoft.Maui.Controls.Compatibility;
+    using Microsoft.Maui.Controls.Hosting;
+    using Microsoft.Maui.Controls.Xaml;
+    using Syncfusion.Maui.Core.Hosting;
+
+    namespace ChartGettingStarted
+    {
+        public static class MauiProgram
+        {
+            public static MauiApp CreateMauiApp()
+            {
+                var builder = MauiApp.CreateBuilder();
+                builder
+                .UseMauiApp<App>()
+                .ConfigureSyncfusionCore()
+                .ConfigureFonts(fonts =>
+                {
+                    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                });
+
+                return builder.Build();
+            }
+        }
+    }
+
+
+{% endhighlight %} 
+
+## Initialize view model
+
+Now, let us define a simple data model that represents a data point on the chart.
+
+{% tabs %}  
+
+{% highlight c# %}
+
+    public class PlantData   
+    {   
+        public string Direction  { get; set; }
+        public double Tree  { get; set; }
+    }
+
+{% endhighlight %} 
+
+{% endtabs %} 
+
+Next, create a view model class and initialize a list of `PlantData` objects as follows.
+
+{% tabs %}  
+
+{% highlight c# %}
+
+    public class ViewModel  
+    {
+        public List<PlantData> PlantDetails { get; set; }      
+
+        public ViewModel()       
+        {
+            PlantDetails  = new List<PlantData>()
+            {
+                new PlantData { Direction = "North", Tree = 80 },
+                new PlantData { Direction = "NorthWest", Tree = 87 },
+                new PlantData { Direction = "West", Tree = 78 },
+                new PlantData { Direction = "SouthWest", Tree = 85 },
+                new PlantData { Direction = "South", Tree = 81 },
+                new PlantData { Direction = "SouthEast", Tree = 88},
+                new PlantData { Direction = "East", Tree = 80},
+                new PlantData { Direction = "NorthEast", Tree = 85}
+            }; 
+        }
+    }
+
+{% endhighlight %} 
+
+{% endtabs %} 
+
+Create a `ViewModel` instance and set it as the chart's `BindingContext`. This enables property binding from `ViewModel` class.
```

Comment:
Create a `ViewModel` instance and set it as the chart's `BindingContext`. This enables property binding from the `ViewModel` class.